src/app/child-dev-project/children/health-checkup/health-checkup-component/health-checkup.component.ts

```diff
@@ -57,11 +42,4 @@ export class HealthCheckupComponent extends RelatedEntitiesComponent<HealthCheck
       return bmi.toFixed(2);
     }
   }
-
-  /**
-   * implements the health check loading from the children service and is called in the onInit()
-   */
-  override getData() {
-    return this.childrenService.getHealthChecksOfChild(this.entity.getId());
```

Comment:
as we rarely use these, I am fine with not using an index here anymore.

However, this makes the HealthCheck module more and more inconsistent, as we only use the index method from ChildrenService for BmiBlock but not here ... 
I would consider removing this from the code base soon. Possibly removing BmiBlock completely for the time being, until we develop a more generic approach for this.

Reply:
Its not an index, its just a `load` and `filter` call on the entity mapper.